spec/web_game_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rspec'
+require 'web_game'
+require 'fake_datastore'
+
+describe WebGame do
```

Comment:
Looks like some tests might be missing for `WebGame`. For example, `save_game`, `start_game`, etc.